src/errorhandler.cpp

```diff
@@ -226,3 +226,12 @@ void ErrorHandler::setAlert(AlertType alert) {
   m_alert = alert;
   emit alertChanged();
 }
+
+// static
+void ErrorHandler::networkErrorHandle(
```

Comment:
nit: `handleNetworkError`? `networkErrorHandle` sounds like the function is going to return a handle to something.

Reply:
Oh I see that `errorHandle` also has "handle" at the end. IMO it also sounds like that function returns a handle to something... But we don't need to make this PR about renaming a bunch of things.